pkg/quota/quota.go

```diff
@@ -33,27 +35,41 @@ type BaseQuota interface {
 func NewQuota(name string) BaseQuota {
 	var quota BaseQuota
 	switch name {
-	case "grp":
+	case "grpquota":
 		quota = &GrpQuota{
 			quotaIDs:    make(map[uint32]uint32),
 			mountPoints: make(map[uint64]string),
 		}
-	case "prj":
+	case "prjquota":
 		quota = &PrjQuota{
 			quotaIDs:    make(map[uint32]uint32),
 			mountPoints: make(map[uint64]string),
 			devLimits:   make(map[uint64]uint64),
 		}
 	default:
-		quota = &GrpQuota{
-			quotaIDs:    make(map[uint32]uint32),
-			mountPoints: make(map[uint64]string),
+		kernelVersion, err := kernel.GetKernelVersion()
+		if err == nil && kernelVersion.Major >= 4 {
```

Comment:
It means that:

> If Linux kernel 4.0+, use project quota, use group quota otherwise.

For this kind of important information, I suggest that adding them to PR description.